web/auth/magic_link.go

```diff
@@ -204,6 +204,12 @@ func magicLinkFlagship(c echo.Context) error {
 		client.ClientID = client.CouchID
 	}
 
+	if err := session.SendNewRegistrationNotification(inst, client.ClientID); err != nil {
+		return c.JSON(http.StatusInternalServerError, echo.Map{
+			"error": err.Error(),
+		})
+	}
+
```

Comment:
Does it make sense to send this new registration notification by e-mail since the magic link itself is sent by e-mail?

Reply:
We are OK with that.